modules/document_repository/README.md

```diff
@@ -0,0 +1,44 @@
+# Document Repository
+
+## Purpose
+
+ The Document Repository is a tool that provides studies with a centralized location for non-candidate documents relevant to the study across multiple candidates.
```

Comment:
Would simplify the first sentence a bit:
"The Document Repository is a tool that provides a centralized location for non-candidate documents relevant to the study. For example, copies of non-filled instruments, publications, manual of operations..."